tmt/steps/prepare/ansible.py

```diff
@@ -35,15 +55,16 @@ class PrepareAnsible(tmt.steps.prepare.PreparePlugin):
     # Supported keys
     _keys = ["playbook", "extra-args"]
 
-    def __init__(self, step, data):
+    def __init__(self, step: str, data: StepDataType) -> None:
```

Comment:
The `step` parameter is an instance of the parent `Step`, not a string.

Reply:
@guy9050 PTAL